### PR TITLE
Add policy-framework-addon logs and configuration yaml output for debug

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Add policy-framework-addon logs and configuration yaml output for debug

### CODE REVIEW

**Review by mprahl — APPROVED** [review]

File: test/e2e/hub_templates_encryption_test.go [hunk 1]
```diff
@@ -300,6 +300,12 @@ func DebugHubEncryption(policyName, secretName, configMapName, hubNs,
 		"ConfigurationPolicy list": {
 			"get", "configurationpolicy", "-n", clusterNamespace,
 		},
+		"ConfigurationPolicy output": {
+			"get", "configurationpolicy", "-n", clusterNamespace, outYaml,
+		},
+		"Governance-policy-framework-addon log": {
+			"logs", "deployment/governance-policy-framework", "-n", clusterNamespace,
```
Comment (mprahl) [inline test/e2e/hub_templates_encryption_test.go:307]: This is only the correct namespace in hosted mode. Are you able to make it work for both?

**Review by mprahl — CHANGES_REQUESTED** [review]

File: test/e2e/hub_templates_encryption_test.go (hunk 1 above)
Comment (yiraeChristineKim) [reply test/e2e/hub_templates_encryption_test.go:307]: I see what you mean! Let me fix Thanks

Comment (yiraeChristineKim) [reply test/e2e/hub_templates_encryption_test.go]: I fixed

**Review by mprahl — APPROVED** [review]

**Pull request merged**